tracks/rust/exercises/pythagorean-triplet/mentoring.md

```diff
@@ -0,0 +1,74 @@
+### Concepts
+
+- Option
+- Math
+
+### Reasonable solutions
+
+A reasonable solution should do the following:
+
+- Use `a.pow(2)` rather than `a*a`
```

Comment:
Because one of the topics on the exercise is math. I always use this as an opportunity to familiarize students with the documentation and get them to look at the pow method on u32 type. I don't disagree with you on the readability issue. But I don't really see what other rust specific math thing the problem writer could have meant so I assumed it was that and just made it a rule to mention it. I'm honestly surprised I put that in the should do category rather than as a bonus. I normally tell students something like "note you could have used pow instead of x*x." But I don't with hold approval over it on it's own.

Reply:
@yaahallo: Would you like to rephrase this so the PR can progress?

Reply:
Sure, I'm not sure exactly how to rephrase this though. It seems like we have a choice between two approaches here, either encourage `u32::pow` or do not. So I can either remove the comment from the mentoring notes and leave it up to the reviewer or I can add the bit about using this exercise to introduce students to math methods to the mentoring notes. I'm fine with either approach, let me know which you prefer.

Reply:
As I read it, the objection is not the use of `pow`, but the recommendation that one *should* do so.

So move the bullet to somewhere that says that you *could* do so.